consensus/ibft/signer/signer.go

```diff
@@ -0,0 +1,336 @@
+package signer
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/0xPolygon/polygon-edge/validators"
+)
+
+var (
+	ErrEmptyCommittedSeals        = errors.New("empty committed seals")
+	ErrEmptyParentCommittedSeals  = errors.New("empty parent committed seals")
+	ErrInvalidCommittedSealLength = errors.New("invalid committed seal length")
+	ErrInvalidCommittedSealType   = errors.New("invalid committed seal type")
+	ErrRepeatedCommittedSeal      = errors.New("repeated seal in committed seals")
+	ErrNonValidatorCommittedSeal  = errors.New("found committed seal signed by non validator")
+	ErrNotEnoughCommittedSeals    = errors.New("not enough seals to seal block")
+	ErrSignerMismatch             = errors.New("mismatch address between signer and message sender")
+	ErrValidatorNotFound          = errors.New("not found validator in validator set")
```

Comment:
Typo here, the error message should be `validator not found in validator set`

Reply:
Fixed at e5aa6f2763d23240dfd84a7bf330ee6d2093fa9d